_posts/2015-07-08-employee-induction-programme.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: "How to welcome new employees"
```

Comment:
Could be seen as arrogant... Like we found the ultimate and only way how to do this. You know what I mean?